helpers/summary-helper.js

```diff
@@ -0,0 +1,55 @@
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
+export class SummaryHelper {
+    static handleSummary(data, environment, testId) {
```

Comment:
Please slice this file to be function based instead spaghetti (you can utilise GPT to slice it and provide functions with names)